### PR TITLE
Upstream nits

### PR DESCRIPTION
**Pull request by oli-obk**
Our lines are often longer than 100 chars. This won't be allowed once miri moves to rustc. I ran rustfmt, which fixed a bunch of those, but there's more (probably inside macros) which rustfmt chokes upon.

### CODE REVIEW

**Comment by oli-obk** [comment]
@eddyb the last two commits are about APFloat integration

**Comment by oli-obk** [comment]
We can undo 145f0c9 and 932a0dd, but we need the rest, since the rest will go into rustc.

I could add an exception to `make tidy` to skip `src/librustc_mir/interpret`, but I don't know how well that'd roll with the rustc devs.

**Comment by eddyb** [comment]
@oli-obk LGTM. You'll have to switch to direct APFloat integration eventually but it's okay for now.

File: src/librustc_mir/interpret/value.rs [hunk 1]
```diff
@@ -154,9 +156,18 @@ pub enum PrimVal {
...
+    I32,
+    I64,
+    I128,
+    U8,
+    U16,
+    U32,
+    U64,
+    U128,
+    F32,
+    F64,
```
Comment (RalfJung) [inline src/librustc_mir/interpret/value.rs]: I think the original grouping here made sense, and it'd be more helpful to keep it.

**Comment by RalfJung** [comment]
I knew I should have hurried to get that other branch I still have lying around locally merged...^^
I will have a gazillion merge conflicts in memory.rs. Could we exempt that file for now and re-rustfmt later? I hope to have the PR ready today or tomorrow (Pacific time).

**Comment by RalfJung** [comment]
I'm going to take the rustup commit here and get it merged independently so that master compiles against nightly again.

**Comment by oli-obk** [comment]
Sure. I'll remove everything but the rustup.

**Comment by oli-obk** [comment]
> I'm going to take the rustup commit here and get it merged independently so that master compiles against nightly again. 

Oh, didn't see that msg. Sure thing!

**Comment by RalfJung** [comment]
Now I am confused, all the other formatting stuff here is still valuable isn't it? In particular the manual line-breaking.

(Though without tidy watching me, I am sure the PRs I send here will break that again...^^)

**Comment by oli-obk** [comment]
Well, those were applied rather quickly. Also we don't need to do those at all in miri, just in librustc_miri

**Pull request closed without merging**